lib/rucio/tests/test_request.py

```diff
@@ -197,6 +198,56 @@ def test_list_requests(self):
         assert len(requests) == 0
 
 
+@pytest.mark.dirty
```

Comment:
I just checked again, and test_request.py was already migrated from unittest to pytest. I would (personally) appreciate if you use pytest for newly added tests. Otherwise it will be me again who'll have to rewrite them :)

Reply:
Sorry but I don't follow. The test `test_list_requests_history` is just a copy/paste of `test_list_requests` with slight amendments

Reply:
You can leave it as is. It's OK this way too.